ocs_ci/ocs/resources/csi_addons.py

```
@@ -231,3 +231,45 @@ def get_csi_addons_config_value(key: str, default: str = "") -> str:
+
+    Raises:
+        AssertionError: If no matching pod is found.
+    """
+    _DRIVER_LABELS = {
+        "cephfs": constants.CSI_CEPHFS_ADDON_NODEPLUGIN_LABEL_420,
+        "rbd": constants.CSI_RBD_ADDON_NODEPLUGIN_LABEL_420,
+    }
+    try:
+        label = _DRIVER_LABELS[driver]
+    except KeyError as exc:
+        raise ValueError(f"Unsupported CSI driver: {driver!r}") from exc
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)